content/blog/gsoc24-improve-ux-notifications-module.rst

```diff
@@ -0,0 +1,241 @@
+Improve UX of the Notifications Module: GSoC’24 Project Report
+==============================================================
+
+:date: 2024-09-16
+:author: Dhanus
+:tags: gsoc
+:category: blog
+:lang: en
+
+.. image:: {static}/images/blog/gsoc-improve-ux-notifications-module/gsoc_2024.jpeg
```

Comment:
please find a way to add the openwisp logo in this image so we have GSoC 24 and OpenWISP together.

Reply:
Done.